TTS/server/server.py

```diff
@@ -75,14 +77,22 @@ def convert_boolean(x):
 if not args.vocoder_config and os.path.isfile(vocoder_config_file):
     args.vocoder_config = vocoder_config_file
 
+
 synthesizer = Synthesizer(args.tts_checkpoint, args.tts_config, args.vocoder_checkpoint, args.vocoder_config, args.use_cuda)
 
+use_speaker_embedding = synthesizer.tts_config.get("use_external_speaker_embedding_file", False)
```

Comment:
the index.html will ask the user to provide 'speaker_json_key'  or  'style_wav' if the model allow multi speaker or gst